src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskList.tsx

```diff
@@ -55,9 +56,9 @@ const TaskList = ({ project }: TaskListProps) => {
   const [addTask, setAddTask] = useState(false);
   const tasks = project.tasks.sort(sortAscendingDate);
 
-  const backLogTasks = tasks.filter((task: Task) => task.status === TaskStatus.IN_BACKLOG);
-  const inProgressTasks = tasks.filter((task: Task) => task.status === TaskStatus.IN_PROGRESS);
-  const doneTasks = tasks.filter((task: Task) => task.status === TaskStatus.DONE);
+  const backLogTasks = tasks.filter((task: Task) => task.status === displayEnum(TaskStatus.IN_BACKLOG));
```

Comment:
you should only use this pipe when the enum is actually being displayed on the frontend. It's being used to calculate values here so you shouldn't use the pipe.

You don't need to find everyplace we're displaying raw enums, but a good example of where to use this pipe is the 'Default Theme' select in settings which is showing the raw enums for its options